src/buildercore/fastly/vcl/journal-submit.vcl.tpl

```diff
@@ -0,0 +1,12 @@
+if (req.url.path == "/submit") {
+    if (req.http.Referer ~ "${referer}") {
+        return(pass);
+    }
+
+    if (randomint(1, 100) <= 10) {
```

Comment:
if we made `10` a template variable, could we ask EJP to immediately configure the redirect to `/submit` and put this to `0` in production?

Reply:
Good idea. Not applied 966fab0dbb82301c659137c8d0d72a5cbc60c5e1 yet, do the values look right?